networkx/algorithms/approximation/broadcasting.py

```diff
@@ -0,0 +1,139 @@
+import networkx as nx
+from networkx import NetworkXError
+from networkx.utils import not_implemented_for
+
+__all__ = [
+    "tree_broadcast_center",
+    "tree_broadcast_time",
+]
+
+
+def _get_max_broadcast_value(G, U, v, values):
+    adj = sorted(set(G.neighbors(v)) & U, key=lambda u: values[u], reverse=True)
+    return max(values[u] + i for i, u in enumerate(adj, start=1))
+
+
+def _get_broadcast_centers(G, v, values, target):
+    adj = sorted(set(G.neighbors(v)), key=lambda u: values[u], reverse=True)
```

Comment:
I don't think you need the `set` function call here.

Reply:
Oh I see, it was needed previously since we were taking the intersection with another set. 
I was just mindlessly copying code from the previous example. 
Thanks for catching this.